oxo.py

```diff
@@ -1,26 +1,73 @@
 class OxoBoard:
+
+    board = {}
+
+    winningCombinations = \
+    [
+        [(0,0),(1,0),(2,0)],
+        [(0,1),(1,1),(2,1)],
+        [(0,2),(1,2),(2,2)],
+        [(0,0),(0,1),(0,2)],
+        [(1,0),(1,1),(1,2)],
+        [(2,0),(2,1),(2,2)],
+        [(0,0),(1,1),(2,2)],
+        [(0,2),(1,1),(2,0)]
```

Comment:
This kind of thing is error-prone and doesn't scale well to larger board sizes. Could this have been initialised programmatically rather than hard-coded?